manifest_staging/deployment/provider-azure-installer.yaml

```diff
@@ -56,8 +57,13 @@ spec:
               drop:
               - ALL
           volumeMounts:
-            - mountPath: "/provider"
-              name: providervol
+            - name: provider-vol
+              mountPath: /provider
+            - name: mountpoint-dir
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: HostToContainer
```

Comment:
Why are we adding this back? This is no longer required with the switch to driver writing secrets.

Reply:
reverted